draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,76 @@ create Evidence. Examples of environments that can be used as
+Attester -- in itself -- is composed and therefore has to conduct the Claims Collection
+in order to create believable Attestation Evidence. This complexity is
+independent of the concept defined by Composite Attesters.
+Composite Attesters are inherently composed of multiple nested environments.
+The following example is intended to illustrate this composition:
+
+A very common example is elaborated on to illustrate Layered Attestation.
+
+The very first Attesting Environment has to ensure the integrity of
+the (U)EFI / BIOS / Firmware that initially boots up a composite device (e.g.
+a cell phone). A hardware component (which effectively can encompasses
+immutable execution environments and mutable execution environments)
+executes the very first Claim Collection. These Claims MUST be stored
+securely (e.g. a eSE. It is not ensured that a TEE or HWM are accessible
+at this very beginning). At this stage of the boot-cycle of a
+thing, the Claims collected typically cannot be composed into Attestation Evidence.
+
+The very first Attesting Environment in this example can be a static root-of-trust
+provided by a hardware component residing next to the (U)EFI / BIOS / Firmware.
+As in any other scenario, this hardware component is is the first Attesting Environment.
+It collects a rather concise number of Claims about the Target Environment.
+The Target Environment in this example is the (U)EFI / BIOS / Firmware.
+After the boot sequence started the Target Environment the
+most important feature of layered attestation is conducted:
+The successfully measured [comment: this is why Henk prefers 'measured' above 'collected'
+because it can be used in English text better] environment that is the
+(U)EFI / BIOS / Firmware now becomes the Attesting Environment.
+
+Now, the duties have been transferred and Layered Attestation takes place.
+The initial Attesting Environment relinquishes its duties to the Target Environment.
+It is important to note that the new Attesting Environment cannot alter
+the content about its own measurements. If the Attesting Environment
+would be able to do that, Layered Attestation would become unfeasible.
+
+In this example the duty of being the Attesting Environment is now
+taken over by the (U)EFI / BIOS / Firmware that was the Attested
+Environment before. This transfer of duty is the essential part of
+Layered Attestation. The (U)EFI / BIOS / Firmware now is the Attesting Environment.
+The next Target Environment is, in this example, a bootloader. There are
+potentially multiple kernels to boot, the decision is imposed on the bootloader.
+Only a integer bootloader will make an appropriate decision. Therefore, Claims about
+the integrity of a bootloader are now collected by the freshly appointed Attesting Environment
+that is the (U)EFI / BIOS / Firmware. Collected Claims have to be stored by the current
+Attesting Environment in a similar shielded and secured manner, so that the next Attesting Environment
```

Comment:
No idea what "shielded" means in this context, but if the claims have been signed then why is that not sufficient?

Reply:
Because I am not sure that every Root of Trust for Measurement does actually sign the measurements. Is that the case? "Shielding" is derived from "shielded location" we should maybe use instead: "A place (memory, register, etc.) where it is safe to operate on sensitive data; data locations that can be accessed only by Protected Capabilities."). Shielded Locations are required to store information, if it is not signed. A can be TEE a shielded location.

Reply:
I assume we have to discuss this, unfortunately